src/lwc/geFormService/geFormService.js

```diff
@@ -42,6 +43,24 @@ class GeFormService {
     fieldTargetMappings;
     donationFieldTemplateLabel;
 
+    getPaymentTransactionStatusEnums = async () => {
+        try {
+            return Object.freeze(
+                JSON.parse(await getPaymentTransactionStatusValues())
```

Comment:
Adding this comment from our discussion this morning for posterity. What do you think about creating an ElevateTransactionStatus javascript class that freezes an object containing hard coded values? I know Javascript and enums don't quite work the same way, but I think my motivation for doing so would moreso be to maintain consistency in matching types between client and server. I'm thinking ideally if we see a type in Apex we know is being used in the UI then we can know there's a corresponding javascript class for that type.

Reply:
@kenlewis-sfdo curious to get your thoughts on this.

Reply:
@dfuller114 @kenlewis-sfdo  - I merged this into the parent branch. Please drop comments into that branch for the full PR review. Thank you both!